### ISSUE TITLE
Error when ~/.config is missing

### ISSUE THREAD

**Issue by mboman** [crunchy-labs/crunchy-cli#149]
**Describe the bug**
Error when ~/.config is missing

**To Reproduce**
Steps / command to reproduce the behavior:
```
❯ ~/src/crunchy-cli/target/release/crunchy-cli --etp-rt COOKIE_CONTENTS_HERE login
:: ✔ Logged in
:: a unexpected error occurred: No such file or directory (os error 2)

```

**Expected behavior**
Either a better error message (example: "~/.config is missing"), or create the missing directory.

**Client (please complete the following information):**
 - OS: CentOS7
 - Version crunchy-cli on  master is 📦 v3.0.0-dev.8 via 🦀 v1.67.1

**Additional context**

**Comment by phrz** [comment]
It would be nice if `crunchy-cli` either had a more clear error message that suggested the need to create the `~/.config` dir, or it just created it when needed.

**Comment by ByteDream** [comment]
Is the issue resolved with the [v3.0.0-dev.9 release](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.0.0-dev.9)?
